cmd/publishing-bot/config/rules.go

```diff
@@ -48,8 +48,8 @@ type BranchRule struct {
 type RepositoryRule struct {
 	DestinationRepository string       `yaml:"destination"`
 	Branches              []BranchRule `yaml:"branches"`
-	// if empty (e.g., for client-go), publisher will use its default publish script
-	PublishScript string `yaml:"publish-script"`
+	SmokeTest             string       `yaml:"smoke-test"` // a multiline bash script
+	Library               bool         `yaml:"library"`
```

Comment:
omitempty?

Reply:
well, default is false. But printing it (if we ever print rules at all ;-) ) makes sense.

Reply:
generating (serializing) is ugly when we default empty fields ;-)

Reply:
fixed